qaoa_vrp/main.py

```diff
@@ -52,62 +52,65 @@ def solve_qaoa(
         prev_optimal_value = 100000
         optimal_value = 1000
 
-        while p <= p_max:
+        while p <= 20:
```

Comment:
Leave this as `p_max` and just generate instances with a large value for `p_max`